### PR TITLE
Config boundaries for word only matching

### PR DESCRIPTION
**Pull request by jeteve**
This adds the ability to specify words boundary when in words only mode. Allows "advanced" usage, such as matching 'C#' in a text.

Additionally, this introduces Qunit tests to improve stability.

### CODE REVIEW

File: jquery.highlight.js [hunk 1]
```diff
@@ -100,7 +113,8 @@
           className: 'highlight',
           element: 'span',
           caseSensitive: false,
-          wordsOnly: false
+            wordsOnly: false,
+            wordsBoundary: '\\b'
```
Comment (knownasilya) [inline jquery.highlight.js]: Can you fix the indentation here, thanks.

**Comment by knownasilya** [comment]
Nice! Thanks for the PR!

**Comment by jeteve** [comment]
Hi! Thx for merging. Any ETA for the next release?

**Comment by knownasilya** [comment]
Within the hour most likely.

**Comment by knownasilya** [comment]
Released as 3.3.0

**Comment by knownasilya** [comment]
Can you submit another PR for documentation in README.md?

**Comment by jeteve** [comment]
Hey Ilya,

Yeah I figured I forgot this bit. I reckon a copy/paste directly in your
master branch will be quicker :)

Cheers,

J.

On 17 August 2015 at 20:59, Ilya Radchenko <notifications@github.com> wrote:

> Can you submit another PR for documentation in README.md?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/knownasilya/jquery-highlight/pull/5#issuecomment-131943956>
> .
>



-- 
Jerome Eteve
+44(0)7738864546
http://www.eteve.net/

**Pull request merged**